test/multi-ns/connection_test.go

```diff
@@ -0,0 +1,348 @@
+	// 				"networkservicemesh.io/app":  "bridge-domain",
+	// 				"networkservicemesh.io/impl": "bridge",
+	// 			},
+	// 		},
+	// 		Template: corev1.PodTemplateSpec{
+	// 			ObjectMeta: metav1.ObjectMeta{
+	// 				Labels: map[string]string{
+	// 					"networkservicemesh.io/app":  "bridge-domain",
+	// 					"networkservicemesh.io/impl": "bridge",
+	// 				},
+	// 			},
+	// 			Spec: corev1.PodSpec{
+	// 				Containers: []corev1.Container{
+	// 					{
+	// 						Name:  "bridge-domain",
+	// 						Image: "networkservicemesh/bridge-domain-bridge:master",
+	// 						Env: []corev1.EnvVar{
+	// 							{
+	// 								Name:  "ENDPOINT_NETWORK_SERVICE",
+	// 								Value: "bridge-domain",
+	// 							},
+	// 							{
+	// 								Name:  "ENDPOINT_LABELS",
+	// 								Value: "app=bridge",
+	// 							},
+	// 							{
+	// 								Name:  "TRACER_ENABLED",
+	// 								Value: "true",
+	// 							},
+	// 							{
+	// 								Name:  "IP_ADDRESS",
+	// 								Value: "10.60.1.0/24",
+	// 							},
+	// 						},
+	// 						Resources: corev1.ResourceRequirements{
+	// 							Limits: map[corev1.ResourceName]resource.Quantity{
+	// 								"networkservicemesh.io/socket": resource.MustParse("1"),
+	// 							},
+	// 						},
+	// 					},
+	// 				},
+	// 			},
+	// 		},
+	// 	},
+	// }
+
+	// dep2 = &appsv1.Deployment{
+	// 	TypeMeta: metav1.TypeMeta{
+	// 		APIVersion: "apps/v1",
+	// 	},
+	// 	ObjectMeta: metav1.ObjectMeta{
+	// 		Name:      "bridge-domain-ipv6",
+	// 		Namespace: "default",
+	// 	},
+	// 	Spec: appsv1.DeploymentSpec{
+	// 		Selector: &metav1.LabelSelector{
+	// 			MatchLabels: map[string]string{
+	// 				"networkservicemesh.io/app":  "bridge-domain-ipv6",
+	// 				"networkservicemesh.io/impl": "bridge-ipv6",
+	// 			},
+	// 		},
+	// 		Template: corev1.PodTemplateSpec{
+	// 			ObjectMeta: metav1.ObjectMeta{
+	// 				Labels: map[string]string{
+	// 					"networkservicemesh.io/app":  "bridge-domain-ipv6",
+	// 					"networkservicemesh.io/impl": "bridge-ipv6",
+	// 				},
+	// 			},
+	// 			Spec: corev1.PodSpec{
+	// 				Containers: []corev1.Container{
+	// 					{
+	// 						Name:  "bridge-domain",
+	// 						Image: "networkservicemesh/bridge-domain-bridge:master",
+	// 						Env: []corev1.EnvVar{
+	// 							{
+	// 								Name:  "ENDPOINT_NETWORK_SERVICE",
+	// 								Value: "bridge-domain-ipv6",
+	// 							},
+	// 							{
+	// 								Name:  "ENDPOINT_LABELS",
+	// 								Value: "app=bridge-ipv6",
+	// 							},
+	// 							{
+	// 								Name:  "TRACER_ENABLED",
+	// 								Value: "true",
+	// 							},
+	// 							{
+	// 								Name:  "IP_ADDRESS",
+	// 								Value: "1200::/120",
+	// 							},
+	// 						},
+	// 						Resources: corev1.ResourceRequirements{
+	// 							Limits: map[corev1.ResourceName]resource.Quantity{
+	// 								"networkservicemesh.io/socket": resource.MustParse("1"),
+	// 							},
+	// 						},
+	// 					},
+	// 				},
+	// 			},
+	// 		},
+	// 	},
+	// }
+)
+
+func TestMain(m *testing.M) {
```

Comment:
For reader clarity I recommend to put `TestMain` into a separate `main_test.go` file. That makes it easy to find and creates clearer distinction since this happens for all tests in package globally, not unlike the usual `main` of programs.